protocol/solid-protocol-test-manifest.ttl

```diff
@@ -27,28 +27,28 @@ manifest:authentication-header
   a td:TestCase ;
     spec:requirementReference sopr:server-authentication,
     sopr:server-unauthenticated ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   spec:testScript
     <https://github.com/solid/specification-tests/protocol/authentication/header.feature> .
 
 manifest:content-negotiation-turtle
   a td:TestCase ;
   spec:requirementReference sopr:server-representation-turtle-jsonld ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   spec:testScript
     <https://github.com/solid/specification-tests/protocol/content-negotiation/content-negotiation-turtle.feature> .
 
 manifest:content-negotiation-jsonld
   a td:TestCase ;
   spec:requirementReference sopr:server-representation-turtle-jsonld ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
```

Comment:
This test and the above: (nit) might want consistency in "Turtle/turtle/TTL" (should be Turtle).